butane_cli/src/main.rs

```diff
@@ -49,6 +49,10 @@ fn main() {
                         .help("Name to use for the migration"),
                 ),
         )
+        .subcommand(
+            clap::Command::new("detachmigration")
```

Comment:
IMO we should be using names like `detach-migration`, but I have followed the existing style.

Reply:
That's fair, it probably would be better naming. Perhaps use Clap's alias to support both? https://docs.rs/clap/latest/clap/struct.Command.html#method.alias